services/catalog/requirements/_base.txt

```diff
@@ -9,7 +9,10 @@ aiodebug==1.1.2
     #   -c requirements/../../../packages/service-library/requirements/./_base.in
     #   -r requirements/../../../packages/service-library/requirements/_base.in
 aiofiles==0.5.0
```

Comment:
this is pretty old, maybe this would be a thing to upgrade in the next maintenance upgrades...